lib/handlebars-code-snippet.js

```diff
@@ -0,0 +1,36 @@
+import handlebars from 'handlebars'
+import hljs from 'highlight.js'
+
+function codeSnippet(language, opts) {
+  if (!opts) {
+    throw new handlebars.Exception('Must pass language to the "code-snippet" helper.')
+  }
+
+  let markup = opts.fn(this)
+
+  // Very basic removal of indentation, reading the amount of spaces in
+  // the first line to remove that amount for every following line
+  let lines = markup.split('\n')
+  let charsToRemove = 0
+  let firstLine = null
+
+  if ((firstLine = lines[0])) {
```

Comment:
Also, what's the community consensus on this kinda coding? I find it a bit weird.